proto/kvrpcpb.proto

```diff
@@ -125,6 +126,7 @@ message KvPair {
     KeyError error = 1;
     bytes key = 2;
     bytes value = 3;
+    bool not_found = 4;
```

Comment:
Seems not useful.

Reply:
I though BatchGet still returned non-existent keys. They are actually filtered. I'll remove it.

Reply:
PTAL again. Thanks.